plugins/woocommerce-admin/client/task-lists/fills/products/stack.scss

```diff
@@ -79,4 +80,22 @@
 		color: $gray-700;
 		line-height: 16px;
 	}
+
+	.woocommerce-stack__overlay-spinner {
+		position: absolute;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		background-color: rgba(255, 255, 255, 0.8);
```

Comment:
We might probably need to consult our designers regarding to styling.

Reply:
That sounds right. Also, I checked the code and noticed we are using HEX codes. So I replaced the background color with this code -
`background-color: #fff;
 opacity: 0.8;`

Reply:
On the other hand, since we've already used the same value for background [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/task-lists/fills/components/load-sample-product-modal.scss#L3), it might be good enough to go ahead with this overlay color.